modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CommerceCatalogLocalServiceImpl.java

```diff
@@ -110,6 +116,29 @@ GroupConstants.DEFAULT_LIVE_GROUP_ID, getLocalizationMap(name),
 		_resourceLocalService.addModelResources(
 			commerceCatalog, serviceContext);
 
+		// CPConfigurationList
+
+		Date date = new Date();
+
+		Calendar calendar = CalendarFactoryUtil.getCalendar(date.getTime());
+
+		int displayDateHour = calendar.get(Calendar.HOUR);
+
+		if (calendar.get(Calendar.AM_PM) == Calendar.PM) {
+			displayDateHour += 12;
+		}
+
+		_cpConfigurationListLocalService.addCPConfigurationList(
+			null, commerceCatalog.getGroupId(), user.getUserId(), 0, true,
+			_language.format(
+				LocaleUtil.fromLanguageId(
+					commerceCatalog.getCatalogDefaultLanguageId()),
+				"master-configuration-x", commerceCatalog.getName(), false),
```

Comment:
mmm... we are saving has name of the list `Master Configuration ${catalogName}`. Is there something that if the catalogName change also the name of the master configurations change?

Reply:
not sure to understand, I've just implemented the mockups